### PR TITLE
Profile: Profile mw8

### PR DESCRIPTION
**Pull request by RhinosF1**

### CODE REVIEW

File: PhpAutoPrepend.php [hunk 1]
```diff
@@ -20,6 +20,19 @@ function wfSetupProfiler() {
 			'output' => 'text',
 		];
 	}
+	$host = isset( $_SERVER['HTTP_HOST'] ) ? $_SERVER['HTTP_HOST'] : '';
```
Comment (Universal-Omega) [inline PhpAutoPrepend.php:23]: $_SERVER['HTTP_HOST'] ?? ''

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Comment by Universal-Omega** [comment]
Actually this entire thing can be simplified by just doing || host === 'mw8...' in the first one, I think.

**Comment by RhinosF1** [comment]
> Actually this entire thing can be simplified by just doing || host === 'mw8...' in the first one, I think.

the config is different

**Comment by Universal-Omega** [comment]
> > Actually this entire thing can be simplified by just doing || host === 'mw8...' in the first one, I think.
> 
> the config is different

I realised that but a few additional checks can be added to make it different for mw8, this is a lot of duplicated code.

File: PhpAutoPrepend.php [hunk 2]
```diff
@@ -20,6 +20,19 @@ function wfSetupProfiler() {
 			'output' => 'text',
 		];
 	}
+	$host = isset( $_SERVER['HTTP_HOST'] ) ? $_SERVER['HTTP_HOST'] ?? '';
```
Comment (Universal-Omega) [inline PhpAutoPrepend.php:23]: $host = $_SERVER['HTTP_HOST'] ?? '';

File: PhpAutoPrepend.php [hunk 3]
```diff
@@ -20,6 +20,19 @@ function wfSetupProfiler() {
...
+	if ( $host = 'mw8.miraheze.org' ) {
```
Comment (Universal-Omega) [inline PhpAutoPrepend.php:24]: $host ===

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**